src/dealDashboard/discussionsList/discussionsList.scss

```diff
@@ -0,0 +1,82 @@
+@import "styles/styles";
+@import "src/resources/elements/primeDesignSystem/styles";
+
+.discussions-list {
+  table {
```

Comment:
much prefer to use divs over tables.  is this solely a semantic thing?

Reply:
replaced to divs.